echopype/convert/set_groups_azfp.py

```diff
@@ -361,7 +361,7 @@ def set_beam(self) -> List[xr.Dataset]:
                 ),
                 "gain_correction": (
                     ["channel"],
-                    unpacked_data["gain"][self.freq_ind_sorted],
```

Comment:
I am curious why `backscatter_r` is set to float32 and `gain_correction` float64. Is this because of what the convention specifies?

Reply:
Heh, it's fairly arbitrary. I was trying to maintain consistency across echopype. `backscatter_r` is float32 in EK60/80 (probably not by design) while `gain_correction` for EK60 is float64. But we could easily change these decisions for AZFP.

The convention generally uses `float` (ie, float32) for most floating-point variables, including these two, whereas echopype typically ends up setting such variables to float64. That's a topic we could revisit in the future.

Reply:
I think we can set all of these to either float32 or float64, for self consistency?

Reply:
Do you mean `backscatter_r` and `gain_correction`, or other variables as well? There are a few other variables here and there that are also set to float32, probably not through explicit actions.

Resetting `backscatter_r` (and `backscatter_i`, for completeness) from float32 to float64 for ek60 and ek80 will likely add to the memory/size footprint (compression probably compensates to some extent). Do you think we should still go ahead and do that? Other than that, I assume there won't be any downstream consequences.